### DISCUSSION TITLE
Set native Avalonia control Background to PrimaryDark

### DISCUSSION THREAD

**Discussion by ohin** [AvaloniaCommunity/Material.Avalonia#267] [Q&A] [answered]
Hello,

I am quite new in Avalonia so I might be missing something about theming and styling.

I don't seem to figure out, how to set for example StackPanel background to Material colors like PrimaryDark in the axaml file. Is it possible to do so? Could anyone please help me?

Using Avalonia 11 and Material.Avalonia 3.0.0-avalonia11-preview5.208-nightly

**Comment by SKProCH** [comment, marked as answer]
Hello, I suppose you need to set `Background="{DynamicResource PrimaryHueDarkBrush}"` to your stack panel. Brush names located here: https://github.com/AvaloniaCommunity/Material.Avalonia/wiki/Brush-Names

**Reply by ohin** [reply]
Oh, perfect, that's it. Thank you very much!
